docs/core/deploying/acquisition-experiences.md

```diff
@@ -0,0 +1,179 @@
+
+While VS Code doesn't come with .NET Core support in the box, adding .NET Core support is straightforward. You can add support in a couple of steps.
+
+1. [Install .NET Core](https://dotnet.microsoft.com/download/dotnet-core)
+2. [Install the C# extension for VS Code from the Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+
+
+## https://dot.net (Microsoft .NET site)
+
+The [Microsoft .NET](https://dot.net/) site is your one stop shop for all things .NET, from [downloads for the latest installers](https://dotnet.microsoft.com/download) for your platform to [Getting Started](https://docs.microsoft.com/dotnet/core/get-started?tabs=windows) guides, and [Tutorials](https://docs.microsoft.com/dotnet/core/tutorials/).
+
+This site provides downloads for the native installers (MSI for Windows, PKG for MacOS), and tar.gz files for download, installation, and redistribution with your app.
+
+
+## Dotnet-install.ps1/.sh Scripts
+
+The dotnet-install scripts are used to perform a non-admin installation of the .NET Core SDK.
+
+The primary usage scenario of these scripts is in automation scenarios and non-admin installations. 
+There are two scripts: one is a PowerShell script that works on Windows and everywhere that PowerShell is supported including [Linux](https://azure.microsoft.com/blog/powershell-is-open-sourced-and-is-available-on-linux/) and [MacOS](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-macos?view=powershell-6). The other is a bash script that works on Linux/MacOS. Both scripts have the same behavior.
+
+For more information about these scripts, see [dotnet-install scripts reference](https://docs.microsoft.com/dotnet/core/tools/dotnet-install-script).
+
+
+## Docker
+
+Containers provide a lightweight way to isolate your application from the rest of the host system, sharing just the kernel, and using resources given to your application.
+.NET Core can run in a Docker container. Official .NET Core Docker images are published to the Microsoft Container Registry (MCR) and are discoverable at the [Microsoft .NET Core Docker Hub repository](https://hub.docker.com/_/microsoft-dotnet-core/). Each repository contains images for different combinations of the .NET (SDK or Runtime) and OS that you can use.
+Microsoft provides images that are tailored for specific scenarios. For example, the [ASP.NET Core repository](https://hub.docker.com/_/microsoft-dotnet-core-aspnet/) provides images that are built for running ASP.NET Core apps in production.
+
+For more information about using .NET Core in a Docker container, see [Introduction to .NET and Docker](https://docs.microsoft.com/dotnet/core/docker/intro-net-docker) and [Samples](https://github.com/dotnet/dotnet-docker/blob/master/samples/README.md).
+
+## Azure App Services
```

Comment:
Is this a sub-item of Docker since it's not listed in the high level list?  Font size is the same which made me think it was a top level item

Reply:
Thanks for the catch.